apps/docs/src/pages/index.module.css

```
@@ -28,13 +28,260 @@ section {
+  color: #343a40;
+}
+
+:global([data-theme="dark"]) .heroCards {
+  color: #dee2e6;
+}
+
+.widgetCard {
+  flex: none;
+  min-width: 0;
+  border-radius: 0.75rem;
+  border: 1px solid color-mix(in srgb, currentColor 12%, transparent) !important;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)